examples/sample-hybrid-project/README.md

```diff
@@ -3,85 +3,68 @@ SAMPLE PROJECT OVERVIEW
-Avail to functions written in Kotlin. This project will be updated over time to 
-reflect changes to Avail as well as to further demonstrate how Avail can be 
-used in a JVM project.
+*This uses alpha version tools that are in flux. This README will be updated* 
+*to reflect accurate usage for the2. 0.0 production release of the Avail* 
+*toolchain. Once the tools are finalized, this sample will be updated to a* 
+*working state relative to the lastest production release.*
+
+This represents a project that uses Avail in a JVM project with Avail dynamic
+pojo linking to functions written in Kotlin. This project will be updated over 
+time to reflect changes to Avail as well as to further demonstrate how Avail can 
+be used in a JVM project.
 
 SETUP
 --------------------------------------------------------------------------------
 The sample project uses Gradle to set up the project environment. It 
 utilizes the `buildSrc` directory for managing state used in `build.gradle.
 kts`.
 
-It uses the [Avail Plugin](https://github.com/AvailLang/gradle-plugin/tree/development) to set up Avail:
+It uses the [Avail Gradle Plugin](../../gradle-plugin/README.md) to set up Avail:
 
 ```kotlin
 id("org.availlang.avail-plugin") version Versions.availGradle
 ```
 
-### Simulate Project Setup
-The directory `roots/` is created by the Avail Gradle task, `initializeAvail` 
-in the task group, `avail`. Delete the `avail` directory and run 
-`initializeAvail` to simulate setting up the Avail project from scratch.
-
-**NOTE** ***This will delete the file,***
-`roots/my-avail-root/App.avail/Foreign Interface.avail`. 
-***This file will need to be added back as it is not code generated by the***
-***Gradle setup. A copy of the file can be found in:*** 
-
-[etc/Foreign Interface.avail](etc/Foreign%20Interface.avail).
+The usage of of the Avail Gradle Plugin is documented extensively in this
+project's [build.gradle.kts](build.gradle.kts) file. Refer to it for plugin 
+usage.
 
-***Simply copy it into:***
+### Project Setup
+As of now, the project is checked into Github already setup. To reset up the
+project "from scratch" do the following:
 
-[roots/my-avail-root/App.avail/](roots/my-avail-root/App.avail)
+1. delete directory `.avail`
+2. delete file `sample-hybrid.json`
+3. Run Gradle task `avail:setupProject`
+```bash
+./gradlew setupProject
+```
 
 FOREIGN FUNCTION INTERFACE
 --------------------------------------------------------------------------------
 
 The foreign function interface is defined in:
 
-[roots/my-avail-root/App.avail/Foreign Interface.avail](roots/my-avail-root/App.avail/Foreign%20Interface.avail)
+[Foreign Interface.avail](roots/my-avail-root/App.avail/Foreign%20Interface.avail)
+
+This is where the bindings are created to the native JVM functionality. To
+dynamically link a JAR, use Avail's `"Link pojos_"`. This is done inside
+Foreign Interface.avail with the statement:
```

Comment:
Let's put `Foreign Interface.avail` inside backticks so that it stands out as a single unit.